notes/SDLC.md

```diff
@@ -0,0 +1,54 @@
+# SDLC 
+## SLDC models: Waterfall vs Agile
```

Comment:
Good titles don't just summarize their children's titles - can you make some introductory statements here before laying out different models?

Reply:
Tried to fix - need more feedback